app/uk/gov/hmrc/agentsubscriptionfrontend/connectors/AgentAssuranceConnector.scala

```diff
@@ -71,6 +71,20 @@ class AgentAssuranceConnector @Inject()(@Named("agent-assurance-baseUrl") baseUr
     }
   }
 
+  def isManuallyAssuredAgent(utr: Utr)(implicit hc: HeaderCarrier): Future[Boolean] = {
+    monitor(s"ConsumedAPI-AgentAssurance-getManuallyAssuredAgents-GET") {
+      val endpoint = s"/agent-assurance/manually-assured/${utr.value}"
+      http.GET[HttpResponse](new URL(baseUrl, endpoint).toString).map{ response =>
+        (200 until 300) contains response.status
```

Comment:
I feel it might not be an efficient method for checking

Reply:
Just double-checked this at http://docs.scala-lang.org/overviews/collections/concrete-immutable-collection-classes.html#ranges : "Ranges are represented in constant space, because they can be defined by just three numbers: their start, their end, and the stepping value. Because of this representation, most operations on ranges are extremely fast."

The contains() function just checks the bounds of the range: https://github.com/scala/scala/blob/07d61ecf134dbba143531f5a1bd3c1289c437296/src/library/scala/collection/immutable/Range.scala#L336

Reply:
that's fine